src/plugins/kibana_utils/common/persistable_state/migrate_to_latest.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { compare } from 'semver';
+import { SerializableState, VersionedState, MigrateFunctionsObject } from './types';
+
+export function migrateToLatest<S extends SerializableState>(
+  migrations: MigrateFunctionsObject,
+  [state, oldVersion]: VersionedState
+): VersionedState<S> {
+  const versions = Object.keys(migrations || {})
+    .sort(compare)
+    .filter((v) => compare(v, oldVersion) > 0);
+
+  if (!versions.length) return [state, oldVersion] as VersionedState<S>;
```

Comment:
you should return currentVersion

Reply:
In this case there are no migrations (`if (!versions.length)`) so there is only one version and "current version" is `oldVersion`.

Reply:
not really, if i have embeddable with version 7.10.0 and i run kibana 7.14.0, migrate to latest should give me embeddable version 7.14.0 even if there are no migrations that needed to run.